src/manifest.json

```diff
@@ -9,42 +9,406 @@
+              }
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "id": "ds-prob",
+          "label": "Probabilistic",
+          "args": {
+            "initialIsOpen": false
+          },
+          "children": [
+            {
+              "type": "internal-link",
+              "id": "ds-prob-intro",
+              "label": "Introduction",
+              "summary": "Probabilistic data structures are very useful for solving streaming analytics problems at scale.",
+              "args": {
+                "path": "/ds/prob/intro.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-hyperloglog",
+              "label": "Cardinality estimation (HyperLogLog)",
+              "args": {
+                "path": "/ds/prob/hyperloglog.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-count-min-sketch",
+              "label": "Frequency estimation (count-min sketch)",
+              "args": {
+                "path": "/ds/prob/count-min-sketch.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-bloom-cuckoo",
+              "label": "Probabilistic set membership (Bloom and cuckoo filters)",
+              "args": {
+                "path": "/ds/prob/bloom-cuckoo.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-top-k",
+              "label": "Probabilistic most frequent items (Top-K)",
+              "args": {
+                "path": "/ds/prob/top-k.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-t-digest",
+              "label": "Percentiles estimation (t-digest)",
+              "args": {
+                "path": "/ds/prob/t-digest.md"
+              }
+            },
+            {
+              "type": "internal-link",
+              "id": "ds-prob-learn-more",
+              "label": "Learn more",
+              "args": {
+                "path": "/ds/prob/learn-more.md"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "type": "group",
+      "id": "sq",
+      "label": "How to query your data",
+      "args": {
+        "initialIsOpen": false
+      },
+      "children": [
+        {
+          "type": "internal-link",
+          "id": "sq-intro",
+          "label": "Introduction",
+          "summary": "Try real-time searching and perform complex structured queries without compromising the database performance.",
+          "args": {
+            "path": "/sq/introduction.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-exact-match",
+          "label": "Exact match (TAG queries)",
+          "args": {
+            "path": "/sq/exact-match.md"
```

Comment:
@ViktarStarastsenka Did you mean to attach an example here?

Reply:
I actually see it attached, but just in case:

![image](https://github.com/RedisInsight/Tutorials/assets/99594890/7656b4f8-6f3e-4744-a7e1-966b3c0e5355)

Reply:
Weird... it didn't show the first time.

This definitely needs Adriano's eyes.

Reply:
as a summary and titles adjustments I will suggest: 

1. **Introduction:** Enable Redis as your main database performing structured queries without compromising your performance and simplicity
2. **Exact Match** (~~TAG queries~~): matching values in your datasets' attributes for secondary index use cases
3. **Full Text Search** (~~fuzzy, wildcard...~~): leveraging advanced text search capabilities such as fuzzy,  wildcard, synonyms, etc...to retrieve the most relevant datasets
4. **Range Queries** (~~numeric~~):  query on numeric fields that returns the values that are in between a given start and end value, such as prices, dates
5. Geospatial Queries (remove):  simple point-based queries and beyond including more complex geometries like polygons. This allows for a more nuanced and detailed querying of spatial data.
6. Vector Search:  efficient retrieval of similar data points in a vector store, enabling advanced applications such as recommendation systems, image and video search, natural language processing, and anomaly detection
7. Combined Queries: combine different query terms and options to create more complex and nuanced queries. 
8. Analytics and transformative: aggregations in Redis allow you you to group, sort, and modify your result data, extracting analytical insights from it. 
9. Semantic Search: pre-filtering using different conditions and predicates combining with generated embeddings, finding similarity between them according with chosen algorithm

Reply:
please align the wording with the "redis voice" @dwdougherty